anitya/templates/api.html

```diff
@@ -1,6 +1,6 @@
 {% extends "master.html" %}
 
-{% block title %} API | Anitya {% endblock %}
+{% block title %} API &mdash; Anitya {% endblock %}
```

Comment:
I'm okay with either one, so unless someone else has very strong feelings about title separators, I'd say pick whatever you like most.